packages/client/package.json

```
@@ -19,6 +19,8 @@
     "prepublishOnly": "npm run build && npm run test",
     "bundle": "webpack",
     "client:start": "ts-node bin/cli.ts",
+    "client:start:dev1": "ts-node bin/cli.ts --discDns=false --discV4=false --bootnodes",
+    "client:start:dev2": "ts-node bin/cli.ts --discDns=false --discV4=false --transports=rlpx --port=30304 --datadir=datadir-dev2",
```

suggestion to dedupe here in case `client:start` changes or gets more complicated

```suggestion
    "client:start:dev1": "npm run client:start -- --discDns=false --discV4=false --bootnodes",
    "client:start:dev2": "npm run client:start -- --discDns=false --discV4=false --transports=rlpx --port=30304 --datadir=datadir-dev2",
```

---

Had this first and then I switched, since it was a bit nicer on the part that the command output is not doubled when invoked, but totally ok here as well, both variants with some not do severe trade offs.